src/components/ToDoItem/ToDoItem.js

```diff
@@ -41,30 +50,48 @@ export const ToDoItem = (message) => {
       return;
     }
     toDoMessage.contentEditable = true;
-    editButton.removeEventListener('mouseout', editButtonHoverOff);
     toDoMessage.focus();
     e.preventDefault();
     editButton.style.opacity = 1;
+    const blurClick = () => {
```

Comment:
change name of this, event is onBlur (means loses focus), not on click. Should be
`handleTaskMessageOnBlur` or smthng similar. Moreover you do not put every method inside separate named function - you can pass below inside addeEventListen anonymous one - () => {}

Reply:
done